src/app/market/sell/add-item/add-item.component.ts

```diff
@@ -239,7 +239,7 @@ export class AddItemComponent implements OnInit, OnDestroy {
         this.templateId = template.id;
         this.log.d('Saved template', template);
         // Need to add location while saving the template
-        this.location.update(this.templateId, country, null, null).subscribe();
+        this.location.execute(locationType.add, this.templateId, country, null, null).subscribe();
```

Comment:
just use 'add', should work.
Same for update.

The types are only to enforce the typing, you can just use the underlying value.